src/Storage/Controllers/DataController.cs

```diff
@@ -287,6 +294,8 @@ public async Task<ActionResult<DataElementList>> GetMany(int instanceOwnerPartyI
 
             DataElement dataElement = await _dataRepository.Create(newData);
             dataElement.SetPlatformSelfLinks(_storageBaseAndHost, instanceOwnerPartyId);
+            
+            await _dataService.PerformFileScan(dataTypeDefinition, dataElement, CancellationToken.None);
```

Comment:
I think we need this for the PUT endpoint as well, in those cases they PUT on an existing data object

Reply:
Added as requested